python/samples/concepts/agents/step1_agent.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+
+from semantic_kernel.agents.chat_completion_agent import ChatCompletionAgent
+from semantic_kernel.connectors.ai.open_ai import AzureChatCompletion
+from semantic_kernel.contents.chat_history import ChatHistory
+from semantic_kernel.contents.utils.author_role import AuthorRole
+from semantic_kernel.kernel import Kernel
+
+###################################################################
+# The following sample demonstrates how to create a simple,       #
+# non-group agent that repeats the user message in the voice      #
+# of a pirate and then ends with a parrot sound.                  #
+###################################################################
+
+# To toggle streaming or non-streaming mode, change the following boolean
+streaming = True
+
+# Define the agent name and instructions
+PARROT_NAME = "Parrot"
```

Comment:
not `AGENT_NAME`?

Reply:
Keeping the naming similar to dotnet, for consistency.